.gitignore

```diff
@@ -17,22 +16,16 @@
 #
 # You should have received a copy of the GNU General Public License version
 # 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
-#
 # ===========================================================================
 #
-# exclude build directories
-/build
-/.build
 # exclude all source directories
 /omr
 /openj9
 /openssl
 # exclude optional eclipse project file
 /.project
```

Comment:
This duplicated entry can be removed as well.

Reply:
No, it seems the unwanted trailing slash was _not_ added in that version (by https://github.com/ibmruntimes/openj9-openjdk-jdk/commit/2306e5b1670400abdcd7786416f63f06e043bd83).